image_setup/01_base_images/build_ros_noetic_20.04.bash

```diff
@@ -9,6 +9,9 @@ if [ $ARCH == "x86_64" ]; then
 elif [ $ARCH == "arm64v8" ]; then
     export BASE_IMAGE=arm64v8/ubuntu:focal
     echo "Building for arm64v8 processor. Using $BASE_IMAGE as base image."
+elif [ $ARCH == "arm32v7" ]; then
```

Comment:
Why not just:
export BASE_IMAGE=${ARCH:+${ARCH}/}ubuntu:focal
if arch is set to "arm64v8" it is "arm64v8/ubuntu:focal" and just "ubuntu:focal" otherwise

Reply:
You are right for making the build on other architectures than the default one more generic and I ll push an update for this in a minute. However for the x86_64 architecture we want to select the nvidia base image. Therefore we have to treat it as an extra case.

Reply:
Normally we have one build script per base image, as often also the package installs and names are different.

Perhaps we can add a parameter in the settings like "use 3D Acceleration"? this would only affect the base image build though, so perhaps the setting should be hidden from "normal users" od the devel images